### ISSUE TITLE
tooltip.y.formatter and yaxis.labels.formatter not working

### ISSUE THREAD

**Issue by nick-keller** [apexcharts/apexcharts.js#46]
On a donut chart (did not check other types) the `tooltip.y.formatter` and `yaxis.labels.formatter` are never called. Therefore values cannot be formated properly.

**Comment by junedchhipa** [comment]
Yes, `tooltip.y.formatter` was not working in donut/pie. Fixed it.
Btw, `yaxis.labels` will only work for axis charts (line/column/area) - not for pie/donuts. 
Thanks for reporting

**Comment by nick-keller** [comment]
Once again awesome work @junedchhipa 👏

**Comment by junedchhipa** [comment]
Here's a [codepen example](https://codepen.io/apexcharts/pen/aaOrjE) using the latest version.

The `tooltip.y.formatter` works as expected. Let me know if I have not misunderstood the main issue.

**Comment by tnusraddinov** [comment]
Tooltip still does not work

**Comment by junedchhipa** [comment]
In the recent updates, I disabled the tooltip for donuts accidentally.
Enabled them again.

For now, you can enable the tooltip by

```javascript
tooltip: {
  enabled: true
}
```
